src/EditorFeatures/Core.Wpf/InlineParameterNameHints/InlineParameterNameHintsTaggerProvider.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.ComponentModel.Composition;
+using Microsoft.CodeAnalysis.Host.Mef;
+using Microsoft.VisualStudio.Text;
+using Microsoft.VisualStudio.Text.Editor;
+using Microsoft.VisualStudio.Text.Tagging;
+using Microsoft.VisualStudio.Utilities;
+
+namespace Microsoft.CodeAnalysis.Editor.InlineParameterNameHints
+{
+    /// <summary>
+    /// The provider that is used as a middleman to create the tagger so that the data tag
+    /// can be used to create the UI tag
+    /// </summary>
+    [Export(typeof(ITaggerProvider))]
+    [ContentType(ContentTypeNames.CSharpContentType)]
```

Comment:
```suggestion
    [ContentType(ContentTypeNames.RoslynContentType)]
```

i.e. ts/js/vb should be able to play here.